### ISSUE TITLE
[A11y]When the focus is on "Provide custom message" text-box, Narrator focus is shifting to "S" letter of "Save" button

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#7893]
Copied from https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1069923
Repro Steps:​
1.Start Screen Reader. Hit the URL "https://www.nuget.org" and sign in with valid credentials​
2.Tab till "Sign In" link in the top navigation and press enter​
3.Tab till "Sign in with Microsoft" button and press enter​
4.Select valid user id and press enter​
5.Enter any Username, tab till "Register" button and press enter​
6.Tab till the user menu button and press enter to expand it. Select "Manage Packages" and press enter​
7.Tab till "Published Packages", expand it and select any existing package.(If no packages are available, add a new package)​
8.Tab till "Manage Package" link under "Info" and press enter​
9.Tab till "Deprecation" section and press enter to expand it​
10.Tab till "Provide custom message" text-box and check if the Narrator focus is on the same​
​
Actual Result:​
When the focus is on "Provide custom message" text-box, Narrator focus is shifting to "S" letter of "Save" button and it is reading as "Provide Custom Message - Edit - Test Deletion - Cap S"
​
NOTE: NVDA is reading "Provide custom message" text box correctly

Expected Result:​
When the focus is on "Provide custom message" text-box, Narrator focus  should be on the same text box and it should only read "Provide Custom Message - Edit - Test Deletion"

Notes:
I was unable to reproduce this. Focus went in to the textbox as expected. Let's verify again that this is behaving as expected and then close it out.

**Comment by ryuyu** [comment]
Again unable to reproduce. Closing
